book/src/package-managers.md

```diff
@@ -101,7 +101,15 @@ Download the official Helix AppImage from the [latest releases](https://github.c
 chmod +x helix-*.AppImage # change permission for executable mode
 ./helix-*.AppImage # run helix
 ```
- 
+
+You can optionally [add the `.desktop` file](./building-from-source.md#configure-the-desktop-shortcut), but there's some pre-requisites:
+```sh
+mkdir -p "$HOME/.local/bin" && \
+mv helix-*.AppImage "$HOME/.local/bin/hx"
```

Comment:
```suggestion
mkdir -p "$HOME/.local/bin"
mv helix-*.AppImage "$HOME/.local/bin/hx"
```

I think it's clearer if these are separate commands

Reply:
I guess? It looks cleaner without `&&`. I added `&&` for safety and performance, but I guess that's irrelevant here, especially considering `mv` should fail anyways if the dir is non-existent

Reply:
yeah that was my thinking as well, if `mkdir` fails then `mv` certainly will too